README.md

```diff
@@ -168,6 +167,8 @@ All properties are optional.
 | labels_secondary | `hover` | `true` / `false` / `hover` | Display secondary Y-axis labels.
 | name_adaptive_color | `false` | `true` / `false` | Make the name color adapt with the primary entity color.
 | icon_adaptive_color | `false` | `true` / `false` | Make the icon color adapt with the primary entity color.
+| grid_line_type | `hour` | `5minute` / `hour` / `day` / `week` | Show grid lines in function of hours_to_show value.
+| grid_line_frequency | 2 |  | Grid lines: thin / thick lines amount ratio.
```

Comment:
Please rename this parameter to "grid_lines_ratio". Also, lets have `grid_lines_type` instead of `grid_line_type` type )))

Reply:
done